### DISCUSSION TITLE
Build error adding dasbob keyboard

### DISCUSSION THREAD

**Discussion by laurapigeon** [manna-harbour/miryoku#273] [Q&A] [answered]
I have [forked the repo](https://github.com/laurapigeon/miryoku_qmk_sturdy) in order to add a new keyboard. I copied across the keyboard folder from [its github](https://github.com/GroooveBob/DASBOB-qmk), added manna-harbour_miryoku from the various other keyboards into the keymaps folder for the new one, and created a new build example based on the others.
I have tested that the Build Example crkbd works within my fork, but when I try the same for the dasbob I get the following error (from [this build](https://github.com/laurapigeon/miryoku_qmk_sturdy/actions/runs/7281288084/job/19841501833)):
```
cc1: all warnings being treated as errors
gmake[1]: *** [builddefs/common_rules.mk:360: .build/obj_dasbob_manna-harbour_miryoku/quantum/keymap_introspection.o] Error 1
gmake: *** [Makefile:392: dasbob:manna-harbour_miryoku] Error 1
Compiling: quantum/keymap_introspection.c                                                           [ERRORS]
 | 
 | 
 | 
Make finished with errors

Error: Process completed with exit code 2.
```

I noticed `keymaps/manna-harbour_miryoku` is absent from crkbd, but without it in my dasbob one I got an error even earlier in the build. This, combined with the fact that the files in the existing manna-harbour_miryoku folders are relatively empty, makes me think I'm not understanding how I need to be adding this new keyboard.

Any help would be greatly appreciated.

**Comment by manna-harbour** [comment, marked as answer]
Dasbob uses a community layout but doesn't have community layout support enabled. See this PR for the single line change you need to make to `info.json` https://github.com/qmk/qmk_firmware/pull/19499. Then you'll be able to build with Miryoku QMK via community layout support https://github.com/manna-harbour/miryoku_qmk/tree/miryoku/users/manna-harbour_miryoku#community-layouts without needing to add anything to any keymap directories.
